packages/core/_src/stm/TMap/operations/indexOf.ts

```diff
@@ -0,0 +1,14 @@
+import { hashUnknown } from "@tsplus/stdlib/structure/Hash"
+
+function hash<K>(k: K): number {
+  const h = hashUnknown(k)
```

Comment:
`Hash` is available globally, so you should do `Hash.unknown` - this way it will automatically resolve to the most specific file when compiling the import statement via TS+.